wgpu/tests/shader_primitive_index/mod.rs

```diff
@@ -51,7 +51,14 @@ fn draw() {
     initialize_test(
         TestParameters::default()
             .test_features_limits()
-            .features(wgpu::Features::SHADER_PRIMITIVE_INDEX),
+            .features(wgpu::Features::SHADER_PRIMITIVE_INDEX)
+            // https://github.com/gfx-rs/wgpu/issues/2751
+            .specific_failure(
+                Some(wgt::Backends::VULKAN),
+                Some(0x1002), // AMD
+                Some("AMD RADV POLARIS12"),
```

Comment:
Let's just match against radv as a whole, it's probably a radv issue more than a Polaris specific issue